HdmiCecSource/HdmiCecSource.cpp

```diff
@@ -787,8 +787,16 @@ namespace WPEFramework
                         powerState = 0; 
                     }
                     else
+		    {
                         powerState = 1;
-
+		    }
+	            if(param->data.state.newState == IARM_BUS_PWRMGR_POWERSTATE_STANDBY)
+                    {
+                        // Handled standby event logic here
+                        isDeviceActiveSource = false;
+                        LOGINFO("ActiveSource isDeviceActiveSource status: %d \n", isDeviceActiveSource);
+                        HdmiCec_2::_instance->sendActiveSourceEvent();
```

Comment:
Hi @kpriya172 , There is no object HdmiCec_2. It is causing compilation failure. Please correct it. Also, please provide the latest log in the Broadcom accelerator device.